src/containers/Home.jsx

```diff
@@ -34,6 +34,23 @@ class Home extends React.Component {
     orgLessUser: false
   }
 
+  handleOrgInviteClick = async (e) => {
+    e.preventDefault()
+    const newInvite = await this.props.mutations.createInvite({
+      id: 'cats',
```

Comment:
do we need these inputs? we're not using them, so worth testing whether everything still works when you remove them